test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -227,8 +102,8 @@ def check_bc(existing_schemas):
     else:
         print(
             "The PR is introducing backward incompatible changes to the "
```

Comment:
Again we can't block developers from making BC-breaking changes in coordination with operators. This test also checks FC, too, which upgraders don't deal with.

Reply:
What does "in coordination with operators" mean? The breaking change is on operator, so it is always in coordination with itself. 

Indeed, we can't deal with FC breaking changes now, it still has to follow 2 week wait policy, we we need at least an allowlist for FC.